src/DatePicker/DatePicker.js

```diff
@@ -284,15 +336,20 @@ class DatePicker extends Component {
     const formatDate = formatDateProp || this.formatDate;
 
     return (
-      <div className={className} style={prepareStyles(Object.assign({}, style))}>
+      <div ref={(node) => this.element = node} className={className} style={prepareStyles(Object.assign({}, style))}>
         <TextField
           {...other}
           onFocus={this.handleFocus}
           onTouchTap={this.handleTouchTap}
+          readOnly={true}
```

Comment:
okay, should it be documented and added as a prop so that it can be passed here?

But I'm not too sure if it makes sense with this control, as allowing the user to overwrite it so that when they press Okay, Cancel or press Enter would be new functionality, in which case should we allow them to type. Currently on a mobile phone they keyboard never pops up, due to the TextField being blur'd upon receiving focus.

Unless I'm missing something, which could very well be the case.